TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -359,20 +359,43 @@ def makeTreeFromMiniAOD(self,process):
     ## ----------------------------------------------------------------------------------------------
     ## Electrons/Muons
     ## ----------------------------------------------------------------------------------------------
+    from TreeMaker.TreeMaker.TMEras import TMeras
     from TreeMaker.Utils.leptonproducer_cfi import leptonproducer
-
-    process.LeptonsNew = leptonproducer.clone(
-        MuonTag          = cms.InputTag('slimmedMuons'),
-        ElectronTag      = cms.InputTag('slimmedElectrons'),
-        PrimaryVertex    = cms.InputTag('offlineSlimmedPrimaryVertices'),
-        minElecPt        = cms.double(10),
-        maxElecEta       = cms.double(2.5),
-        minMuPt          = cms.double(10),
-        maxMuEta         = cms.double(2.4),
-        UseMiniIsolation = cms.bool(True),
-        muIsoValue       = cms.double(0.2),
-        elecIsoValue     = cms.double(0.1), # only has an effect when used with miniIsolation
-        METTag           = METTag, 
+    process.LeptonsNew = leptonproducer.clone()
+    TMeras.TM2016.toModify(process.LeptonsNew,
+        elecIsoValue       = cms.double(0.1), # only has an effect when used with miniIsolation
```

Comment:
still some duplicates here

Reply:
Yeah, for both this comment and the previous something weird happened. I made the change, but it didn't get synchronized to the server. Will be in the next commit.